frambo/config.py

```diff
@@ -52,11 +54,16 @@ def frambo_config(cfgdir=CONFIG_DIR):
     return config
 
 
-def get_from_frambo_config(module, key):
-    value = frambo_config().get(module, {}).get(key)
-    if not value:
+def get_from_frambo_config(
+    module: str,
+    key: str,
+    default: Any = "",
+    raises: bool = True,
+) -> str:
```

Comment:
I think the return value can be any json supported object
```suggestion
) -> Any:
```

Reply:
Good catch. I can be.